tkintertools/theme/Button/Information.dark.json

```diff
@@ -1,11 +1,11 @@
-{
-    "normal": {
-        "fill": "#F1F1F1"
-    },
-    "hover": {
-        "fill": "#F1F1F1"
-    },
-    "active": {
-        "fill": "#F1F1F1"
-    }
+{
+    "normal": {
+        "fill": "#ffffff"
+    },
+    "hover": {
+        "fill": "#ffffff"
+    },
+    "active": {
+        "fill": "#ffffff"
+    }
 }
```

Comment:
Maybe you should consider using uppercase color strings? I think it's eye-catching and pretty.